pkg/addons/embeddedclusteroperator/embeddedclusteroperator.go

```diff
@@ -48,6 +49,21 @@ var helmValues = map[string]interface{}{
 	"utilsImage":                UtilsImage,
 }
 
+func init() {
+	if ImageOverride != "" {
+		// split ImageOverride into the image and tag
+		parts := strings.Split(ImageOverride, ":")
```

Comment:
What if one uses an image reference by `sha` ? 

Can we not `panic` ? I mean, I guess we can assess this later on and return an error instead.

Reply:
this is only for dev use
if you are compiling in an image that won't work with this, I think panicking in the build process in CI is appropriate (this will cause a panic when getting version metadata, which happens in CI)

Reply:
Ah, I see. This will explode upon calling `embedded-cluster version metadata` AND it is intended only for whoever knows what they are doing. OK. LGTM.